src/index.js

```diff
@@ -197,9 +199,10 @@ async function run(octokit, context, token) {
 		let commentId;
 		try {
 			const comments = (await octokit.issues.listComments(commentInfo)).data;
+			const commentRegExp = new RegExp(`<sub>[\s\n]*(compressed|gzip)-size-action${commentKey ? `::${commentKey}` : ''}`)
```

Comment:
I'd also include the closing `</sub>` tag in the RegExp otherwise an action without a comment-key could match/update another action with a comment-key?

I didn't run the PR but that looks good to me overall.

Reply:
Good call

```suggestion
			const commentRegExp = new RegExp(`<sub>[\s\n]*(compressed|gzip)-size-action${commentKey ? `::${commentKey}` : ''}</sub>`)
```